lib/ui/tabs/event_page.dart

```diff
@@ -122,6 +122,10 @@ class EventDetailPageState extends State<EventDetailPage> {
                             },
                             child: CachedNetworkImage(
                               imageUrl: doc.get('image'),
+                              errorWidget: (context, url, error) => const Icon(
+                                Icons.broken_image,
+                                color: Colors.grey,
```

Comment:
This doesn't look so good, update.this with something else.

Reply:
Should I increase the size of the Icon?

Reply:
No use a proper placeholder image instead, and it's color shouldn't be too light otherwise the back button may be hidden

Reply:
How about this?
![image](https://user-images.githubusercontent.com/53429438/94772964-770b9880-03d8-11eb-9649-ad6cf883d7a1.png)

Reply:
Seems fine, just increase the icon size

Reply:
Changes made